views/components/form-fields-next/macros.njk

```diff
@@ -152,7 +152,7 @@
 
     <select class="ff-select" id="field-{{ field.name }}" name="{{ field.name }}">
         {% if field.defaultOption %}
-            <option disabled value{% if not field.value %} selected{% endif %}>{{ field.defaultOption }}</option>
```

Comment:
This fixes a bug when changing country where the first item in the list would get selected, rather than defaulting to "select a country"